opcua/common/uaerrors/_auto.py

```diff
@@ -0,0 +1,596 @@
+#AUTOGENERATED!!!
```

Comment:
other files are names ***_auto.py, so maybe we should name this file uaerrors_auto.py. or rename the others if _xx is common convention.